modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerConsistencyTest.java

```diff
@@ -297,9 +296,9 @@ public void testRepairNonExistentCache() throws Exception {
         for (int i = 0; i < PARTITIONS; i++) {
             assertEquals(EXIT_CODE_UNEXPECTED_ERROR,
                 execute("--consistency", "repair",
-                    ConsistencyCommand.CACHE, "non-existent",
-                    ConsistencyCommand.PARTITIONS, String.valueOf(i),
-                    ConsistencyCommand.STRATEGY, strategy.toString()));
+                    KillCommandsControlShTest.CACHE, "non-existent",
```

Comment:
I suppose it would be more clear if you define these constants in this test class.